### PR TITLE
chore(cli): combine hostname and path

### PR DESCRIPTION
**Pull request by huanjani**
For the 'ecs-preview app show' command, I have combined the output that was previously 'URL' and 'Path' into a combined 'URL'. 

My next step will be to add the protocol and make it clickable.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: templates/custom-resources/alb-rule-priority-generator.js [hunk 1]
```diff
@@ -0,0 +1 @@
...:e});const l=new j(a.ResponseURL||defaultResponseURL),m={hostname:l.hostname,port:443,path:l.pathname+l.search,method:"PUT",headers:{"Content-Type":"","Content-Length":k.length}};i.request(m).on("error",h).on("response",a=>{a.resume(),400<=a.statusCode?h(new Error(`Error ${a.statusCode}: ${a.statusMessage}`)):g()}).end(k,"utf8")})};const calculateNextRulePriority=async function(a){var b,c=new aws.ELBv2,d=[];do{const e=await c.describeRules({ListenerArn:a,Marker:b}).promise();d=d.concat(e.Rules),b=e.NextMarker}while(b);let e=1;if(0<d.length){const a=d.map(a=>"default"===a.Priority?0:parseInt(a.Priority)),b=Math.max(...a);e=b+1}return e};exports.nextAvailableRulePriorityHandler=async function(a,b){var c,d,e={};try{switch(a.RequestType){case"Create":d=await calculateNextRulePriority(a.ResourceProperties.ListenerArn),e.Priority=d,c=`alb-rule-priority-${a.LogicalResourceId}`;break;case"Update":case"Delete":c=a.PhysicalResourceId;break;default:throw new Error(`Unsupported request type ${a.RequestType}`);}await report(a,b,"SUCCESS",c,e)}catch(d){console.log(`Caught error ${d}.`),await report(a,b,"FAILED",c,null,d.message)}},exports.withDefaultResponseURL=function(a){defaultResponseURL=a};
```
Comment (iamhopaul123) [inline templates/custom-resources/alb-rule-priority-generator.js]: Umm these files should exist if you do any of "make build" or "make test" correctly before committing.

File: templates/custom-resources/dns-cert-validator.js [hunk 2]
```diff
@@ -0,0 +1 @@
...ordSet:{Name:d,Type:e,TTL:60,ResourceRecords:[{Value:f}]}}]},HostedZoneId:b}).promise()},clients=function(a){const b=new aws.ACM({region:a}),c=new aws.Route53;return waiter&&(c.waitFor=b.waitFor=waiter),[b,c]};exports.certificateRequestHandler=async function(a,b){var c,d,e={};try{switch(a.RequestType){case"Create":case"Update":d=await requestCertificate(a.RequestId,a.ResourceProperties.DomainName,a.ResourceProperties.SubjectAlternativeNames,a.ResourceProperties.HostedZoneId,a.ResourceProperties.Region),e.Arn=c=d;break;case"Delete":c=a.PhysicalResourceId,c.startsWith("arn:")&&(await deleteCertificate(c,a.ResourceProperties.Region,a.ResourceProperties.HostedZoneId));break;default:throw new Error(`Unsupported request type ${a.RequestType}`);}await report(a,b,"SUCCESS",c,e)}catch(d){console.log(`Caught error ${d}.`),await report(a,b,"FAILED",c,null,d.message)}},exports.withDefaultResponseURL=function(a){defaultResponseURL=a},exports.withWaiter=function(a){waiter=a},exports.withSleep=function(a){sleep=a},exports.reset=function(){sleep=defaultSleep,random=Math.random,waiter=void 0,maxAttempts=10},exports.withRandom=function(a){random=a},exports.withMaxAttempts=function(a){maxAttempts=a};
```
Comment (iamhopaul123) [inline templates/custom-resources/dns-cert-validator.js]: same

File: templates/custom-resources/dns-delegation.js [hunk 3]
```diff
@@ -0,0 +1 @@
...eResourceRecordSets({ChangeBatch:{Changes:[recordChangeAction("DELETE",c,"NS",j.ResourceRecords)]},HostedZoneId:h}).promise();return await waitForRecordSetChange(e,k.ChangeInfo.Id),c},recordChangeAction=function(a,b,c,d){return{Action:a,ResourceRecordSet:{Name:b,Type:c,TTL:60,ResourceRecords:d}}},waitForRecordSetChange=function(a,b){return a.waitFor("resourceRecordSetsChanged",{$waiter:{delay:30,maxAttempts:10},Id:b}).promise()};exports.domainDelegationHandler=async function(a,b){var c,d={};try{switch(a.RequestType){case"Create":case"Update":const b=await createSubdomainInRoot(a.RequestId,a.ResourceProperties.DomainName,a.ResourceProperties.SubdomainName,a.ResourceProperties.NameServers,a.ResourceProperties.RootDNSRole);d.Arn=c=b;break;case"Delete":await deleteSubdomainInRoot(a.RequestId,a.ResourceProperties.DomainName,a.ResourceProperties.SubdomainName,a.ResourceProperties.RootDNSRole),c=a.PhysicalResourceId;break;default:throw new Error(`Unsupported request type ${a.RequestType}`);}await report(a,b,"SUCCESS",c,d)}catch(d){console.log(`Caught error ${d}.`),console.log(d),await report(a,b,"FAILED",c,null,d.message)}},exports.withDefaultResponseURL=function(a){defaultResponseURL=a};
```
Comment (iamhopaul123) [inline templates/custom-resources/dns-delegation.js]: same

File: templates/packrd/packed-packr.go [hunk 4]
```diff
@@ -0,0 +1,57 @@
+// +build !skippackr
```
Comment (iamhopaul123) [inline templates/packrd/packed-packr.go]: same I wonder where this change comes from

File: templates/templates-packr.go [hunk 5]
```diff
@@ -0,0 +1,8 @@
+// +build !skippackr
```
Comment (iamhopaul123) [inline templates/templates-packr.go]: same

File: internal/pkg/cli/app_show_test.go [hunk 6]
```diff
@@ -476,7 +476,7 @@ func TestAppShow_Execute(t *testing.T) {
...
+			wantedContent: "{\"appName\":\"my-app\",\"type\":\"\",\"project\":\"my-project\",\"configurations\":[{\"environment\":\"test\",\"port\":\"80\",\"tasks\":\"1\",\"cpu\":\"256\",\"memory\":\"512\"},{\"environment\":\"prod\",\"port\":\"5000\",\"tasks\":\"3\",\"cpu\":\"512\",\"memory\":\"1024\"}],\"routes\":[{\"environment\":\"test\",\"url\":\"my-pr-Publi.us-west-2.elb.amazonaws.com/frontend\"},{\"environment\":\"prod\",\"url\":\"my-pr-Publi.us-west-2.elb.amazonaws.com/backend\"}],\"variables\":[{\"environment\":\"prod\",\"name\":\"ECS_CLI_ENVIRONMENT_NAME\",\"value\":\"prod\"},{\"environment\":\"test\",\"name\":\"ECS_CLI_ENVIRONMENT_NAME\",\"value\":\"test\"}],\"resources\":{\"prod\":[{\"type\":\"AWS::EC2::SecurityGroupIngress\",\"physicalID\":\"ContainerSecurityGroupIngressFromPublicALB\"}],\"test\":[{\"type\":\"AWS::EC2::SecurityGroup\",\"physicalID\":\"sg-0758ed6b233743530\"}]}}\n",
```
Comment (efekarakus) [inline internal/pkg/cli/app_show_test.go]: looks like a fun change 😅

File: templates/cicd/buildspec.yml [hunk 7]
```diff
@@ -1,61 +0,0 @@
-# Buildspec run in the build stage of your pipeline.
```
Comment (efekarakus) [inline templates/cicd/buildspec.yml]: We probably shouldn't delete this file :) The PR doesn't seem to need touching any of the files under the "templates" directory

Comment (huanjani) [reply templates/cicd/buildspec.yml]: TY!

File: templates/custom-resources/alb-rule-priority-generator.js (hunk 1 above)
Comment (huanjani) [reply templates/custom-resources/alb-rule-priority-generator.js]: > Omg this is such a great commit! Really appreciate this change, it's gonna be so good.
> 
> A couple of small things:
> 
> 1. Can you un-commit the templates/* files? They're generated at build time so we don't need to commit them 😁
> 2. Can you also update the End to End Tests ( e2e/*) and find any code that looks like: `			resp, fetchErr := http.Get(fmt.Sprintf("http://%s/%s/", app.Routes[0].URL, app.Routes[0].Path))` and have it just use the URL?
> 3. Also, just an FYI, we use [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) for our git commits so we can automatically generate clean release notes :D  I think PH update this PR, but just a future reference!
> 
> Thank you!

Thank you! Should I put `templates/` into the .gitignore file?

Comment (iamhopaul123) [reply templates/custom-resources/alb-rule-priority-generator.js]: No you don't. Those files in `templates/` should not be changed.

Comment (huanjani) [reply templates/custom-resources/alb-rule-priority-generator.js]: Ah, got it. Now I see what happened. Thanks!

File: e2e/init/init_test.go [hunk 8]
```diff
@@ -83,7 +83,7 @@ var _ = Describe("init flow", func() {
 			Expect(len(app.Routes)).To(Equal(1))
 			Expect(app.Routes[0].Environment).To(Equal("test"))
 			Expect(app.Routes[0].Path).To(Equal(appName))
-			resp, fetchErr := http.Get(fmt.Sprintf("http://%s/%s/", app.Routes[0].URL, app.Routes[0].Path))
+			resp, fetchErr := http.Get(fmt.Sprintf("http://%s/", app.Routes[0].URL))
```
Comment (efekarakus) [inline e2e/init/init_test.go]: I believe the other e2e tests "multi-app-project" and "multi-env-project" need to be updated similarly

**Pull request merged**